### PR TITLE
Breadcrumb #428

### PR DESCRIPTION
**Pull request by manndave**
Fix Issue #428 - allow breadcrumb to show when less than maxitems in list.

### CODE REVIEW

**Comment by mikewheaton** [comment]
Tested it with only 3 items (on master that results in a blank page) and it's working beautifully:
image

Approved by me. We just need one more reviewer before merging it in.

File: src/components/Breadcrumb/Breadcrumb.js [hunk 1]
```diff
@@ -119,7 +119,7 @@ fabric.Breadcrumb.prototype = (function() {
   var _addBreadcrumbItems = function(maxItems) {
     _resetList(_breadcrumbList);
     var i = _itemCollection.length - maxItems;
-
+    i=i<0? 0 : i;
```
Comment (wdo3650) [inline src/components/Breadcrumb/Breadcrumb.js]: Nit. That'd be great if you could put spaces between the variables, values and operators in the first part of the ternary operation.

**Comment by manndave** [comment]
Client stuff keeping me OOO for the rest of the week, but I'll do this ASAP and re-push the PR.  Thanks

**Comment by manndave** [comment]
New PR posted...

File: src/components/Breadcrumb/Breadcrumb.js [hunk 2]
```diff
@@ -119,7 +119,7 @@ fabric.Breadcrumb.prototype = (function() {
   var _addBreadcrumbItems = function(maxItems) {
     _resetList(_breadcrumbList);
     var i = _itemCollection.length - maxItems;
-    i=i<0? 0 : i;
+    i = i < 0? 0 : i;
```
Comment (wdo3650) [inline src/components/Breadcrumb/Breadcrumb.js]: Thanks for making these changes. Nit, can you add a space between the 0 and the ?.

**Comment by manndave** [comment]
New PR posted

**Comment by wdo3650** [comment]
@manndave Awesome, thanks for the contribution!

**Pull request merged**